src/shogun/distance/LevensteinDistance.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Yuhui Liu
+ *
+ */
+#ifndef _LEVENSTEINDISTANCE_H__
+#define _LEVENSTEINDISTANCE_H__
+
+#include <string>
+#include <vector>
+namespace shogun
+{
+	class LevensteinDistance
```

Comment:
This class should probably inherit from Distance no?

Reply:
I find `Distance` have many api that LevensteinDistance don't need, such as `virtual bool init(std::shared_ptr<Features> lhs, std::shared_ptr<Features> rhs);`. Because the function of LevensteinDistance is that compare two class names which is `string`.

Reply:
hi, @gf712, Can you give me some idea about implement Levenshtein algorithm for shogun Features?

Reply:
Hi @gf712 , I have changed LevenshteinDistance inherit from Distance, Please have a look when you are free, thanks :)